### PR TITLE
[enterprise-logs] Add host aliases

### PR DESCRIPTION
**Pull request by YuleZ**
I need to set hostAliases in my environment for loki-distributed.
This PR adds this feature to the other helm-charts as well as for loki-distributed where I need this change initially.

### CODE REVIEW

File: charts/enterprise-logs/values.yaml [hunk 1]
```diff
@@ -344,6 +359,11 @@ gateway:
 compactor:
   # -- Define the amount of instances
   replicas: 1
+  # -- hostAliases to add
+  hostAliases: [ ]
```
Comment (zanhsieh) [inline charts/enterprise-logs/values.yaml:363]:
```suggestion
  hostAliases: []
```

File: charts/enterprise-logs/values.yaml [hunk 2]
```diff
@@ -270,6 +280,11 @@ adminApi:
 gateway:
   # -- Define the amount of instances
   replicas: 1
+  # -- hostAliases to add
+  hostAliases: [ ]
```
Comment (zanhsieh) [inline charts/enterprise-logs/values.yaml:284]:
```suggestion
  hostAliases: []
```

File: charts/enterprise-logs/values.yaml [hunk 3]
```diff
@@ -201,6 +206,11 @@ tokengen:
 adminApi:
   # -- Define the amount of instances
   replicas: 1
+  # -- hostAliases to add
+  hostAliases: [ ]
```
Comment (zanhsieh) [inline charts/enterprise-logs/values.yaml:210]:
```suggestion
  hostAliases: []
```

File: charts/enterprise-logs/values.yaml [hunk 4]
```diff
@@ -172,6 +172,11 @@ structuredConfig: {}
 tokengen:
   # -- Whether the job should be part of the deployment
   enable: true
+  # -- hostAliases to add
+  hostAliases: [ ]
```
Comment (zanhsieh) [inline charts/enterprise-logs/values.yaml:176]:
```suggestion
  hostAliases: []
```

**Review by zanhsieh — CHANGES_REQUESTED** [review]

**Comment by YuleZ** [comment]
@zanhsieh can you check what's wrong with the lint-test job? Looks like a connectivity issue inside the test. I don't think this relate to my changes.

**Comment by zanhsieh** [comment]
@YuleZ 
I did try to figure it out but still puzzling. No idea why helm add minio repo would fail.

**Comment by zanhsieh** [comment]
@YuleZ Can you rebase this PR please? It seems this PR does not recognize ct.yaml change.

**Review by zanhsieh — APPROVED** [review]

**Pull request merged**